### DISCUSSION TITLE
Kata solution not seen by me

### DISCUSSION THREAD

**Discussion by seshoumara** [codewars/codewars.com#3165] [Q&A] [answered]
Hello. I solved the https://www.codewars.com/kata/550f22f4d758534c1100025a kata in C++.

But when I go to Solutions -> Mine there's nothing there, showing me "No solutions". I wonder if it's my PC and if so what could it possibly be? All other solutions show fine though.

**Comment by hobovsky** [comment]
It happens sometimes. Mods can try to force the solution to appear, but we need to know your profile, and it does not always work.

**Comment by seshoumara** [comment]
What do you mean by knowing my profile? One can already see my site name. A link perhaps?

**Reply by hobovsky** [reply]
How can I see your site name in this discussion? I can't, I can see only your github name.

**Reply by seshoumara** [reply]
Sorry, I assumed it's visible. My site name is seshoumara, profile at https://www.codewars.com/users/seshoumara

**Reply by hobovsky** [reply, marked as answer]
I tried to make your solution of "Directions Reduction" visible, and I think it worked.

**Reply by seshoumara** [reply]
It worked. Thank you!
